Makefile.in

```diff
@@ -0,0 +1,870 @@
+# Makefile.in generated by automake 1.15 from Makefile.am.
```

Comment:
There's a bunch of these generated files in this pull request clogging up the changes, and these shouldn't be in a pull request anyway. I think these are already in the .gitignore, but if they aren't on your repo you need to add them back.

Reply:
These makefiles won't be cleaned up by a rebase, these generated files were never in master to begin with so a rebase will never tell your branch to delete them. We could add some lines to the gitignore, or we can go through and clean out all the generated makefiles in your repo. A hefty amount of the junk in the changed files view is actually these files, so your aggregate changeset probably gets significantly easier to review after we don't need to scroll past all these identical files.